index.html

```diff
@@ -6,9 +6,37 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>portfolio</title>
 
+    <!-- Bootstrap icons link -->
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.2/font/bootstrap-icons.css"
+    />
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <h1 class="header">"Hello!"</h1>
+    <header class="header">
+      <nav class="navbar">
+        <div class="logo">BZ.</div>
+        <i class="bi bi-list"></i>
+      </nav>
+
+      <section class="hero">
+        <div class="hero__title">
+          <h1>
+            I'm Bahoz<br />
+            Glad to see<br />
+            you!
+          </h1>
+        </div>
+        <p class="hero__text">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and <br />
+          experience! If you like what you see and have a project you need
+          coded, don’t hestiate to contact me.
+        </p>
+        <p class="hero__connect">LET’S CONNECT</p>

```

Comment:
- Please use an `<a>` tag here, as it is going to be an on-page link to your contact form in the future phase of this project.
- Please make your readme customised to this project and edit the sections in the readme to suit this project, so as to make your project more professional/